iothub/service/src/Common/Security/CryptoKeyGenerator.cs

```diff
@@ -73,6 +83,10 @@ public static string GenerateKeyInHex(int keySize)
             return BitConverter.ToString(keyBytes).Replace("-", "");
         }
 
+        /// <summary>
+        /// Generate a unique GUID.
```

Comment:
I still feel this isn't specific enough. It creates a special kind of GUID, right? Can we delve into that?

Reply:
oops, I changed the '<return>' comment, but not the summary one. Changing now.